mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -6850,6 +6892,50 @@ resource "google_compute_instance" "foobar" {
 `, instance, enableConfidentialCompute)
 }
 
+<% unless version == 'ga' -%>
+func testAccComputeInstanceConfidentialHyperDiskBootDisk(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_compute_image" "my_image" {
+  family    = "ubuntu-2204-lts"
+  project   = "ubuntu-os-cloud"
+}
+
+data "google_project" "project" {}
+
+resource "google_kms_crypto_key_iam_binding" "crypto_key" {
```

Comment:
The test should use BootstrapPSARole (or one of its friends) to ensure that this permission is not removed from the service account for the main test project.